plugins/inputs/phpfpm/phpfpm.go

```diff
@@ -277,6 +278,68 @@ func importMetric(r io.Reader, acc telegraf.Accumulator, addr string) (poolStat,
 	return stats, nil
 }
 
+func expandUrls(urls []string) ([]string, error) {
+	addrs := make([]string, 0, len(urls))
+	for _, url := range urls {
+		if isNetworkURL(url) {
+			addrs = append(addrs, url)
+			continue
+		}
+		paths, err := globUnixSocket(url)
+		if err != nil {
+			return nil, err
+		}
+		addrs = append(addrs, paths...)
+	}
+	return addrs, nil
+}
+
+func globUnixSocket(url string) ([]string, error) {
+	pattern, status := unixSocketPaths(url)
+	paths, err := filepath.Glob(pattern)
+	if err != nil {
+		return nil, err
```

Comment:
might be good to add some context here. 
```suggestion
		return nil, errors.Wrapf(err, "couldn't read the file path pattern %q", pattern)
```

Reply:
I didn't add any context because `filepath.Glob` can only fail due to invalid syntax in the pattern, but I guess it's useful to add the bad pattern in the error message, especially if multiple patterns are used.